### ISSUE TITLE
Not working with gradle 3.1.3

### ISSUE THREAD

**Issue by AvivVegh** [NicoToast/fat-aar#2]
Hi this repo is not working on 3.1.3 
can you help ?

**Comment by NicoToast** [comment]
Yes, this version of the plug-in cannot be working in versions above 3.1.0.
This is because the packaging structure has changed so much since 3.1.0 that the "build/intermediates/exploded-aar" folder created in the middle has disappeared.
So there is no effective solution until now.

**Comment by jessandroid** [comment]
this repo is not working on gradle plugin 3.1.3
And I have got this error message :
* What went wrong:
Execution failed for task ':libraryModule:generateReleaseRFile'.
> java.util.NoSuchElementException (no error message)

can you help ?

**Comment by mo3in** [comment]
please upgrade to gradle 3.1.3

**Comment by jgmGit** [comment]
Waiting for gradle 3.1.3 support
